src/main/java/com/google/api/generator/gapic/composer/grpc/MockServiceImplClassComposer.java

```diff
@@ -97,12 +90,34 @@ public static MockServiceImplClassComposer instance() {
   }
 
   @Override
-  public GapicClass generate(GapicContext ignored, Service service) {
+  public GapicClass generate(GapicContext context, Service service) {
     TypeStore typeStore = createDynamicTypes(service);
     String className = ClassNames.getMockServiceImplClassName(service);
     GapicClass.Kind kind = Kind.TEST;
     String pakkage = service.pakkage();
 
+    // Use the full name java.lang.Object if there is a proto message that is also named "Object".
+    // Affects GCS.
+    if (context.messages().keySet().stream()
+        .map(s -> s.substring(Math.max(0, s.lastIndexOf(".") + 1), s.length()))
+        .collect(Collectors.toSet())
```

Comment:
Looks like you don't really need the collected set of stuff, you just curious if three is any object of name "Object", so please consider something shorter, like:
```java
if(context.messages().keySet().stream().anyMatch(s -> s.endsWith(".Object"))) {}
```

Reply:
Done.